.github/workflows/pull-legal-docs.yml

```diff
@@ -0,0 +1,50 @@
+name: Pull legal-docs
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * TUE'
+
+jobs:
+  pull-legal-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone FxA code repository
+        uses: actions/checkout@v3
+        with:
+          path: fxa
+          fetch-depth: 2
+
+      - name: Clone legal-docs repository
+        uses: actions/checkout@v3
+        with:
+          repository: mozilla/legal-docs
+          ref: prod
+          path: legal-docs
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think you need the secret to pull from legal-docs (public repo)?

Reply:
That's what I was expecting as well, but I kept running into errors similar to the one in [this failed run](https://github.com/mozilla/fxa/actions/runs/3274794590/jobs/5388794337).

Reply:
Weird. Not sure if the workflow link is correct in that page, but the workflow has a `token` attribute. I assume the error means "you provide a token attribute but it's not in the repo's settings" more than "you need to provide a token".

With that said, if it works, I probably wouldn't spend more time on it. Debugging actions is not fun.